scripts/common/longhorn.sh

```diff
@@ -207,7 +215,14 @@ function longhorn_prepare_for_migration() {
     logFail "During the preparation for Longhorn, some replicas may have been scaled down to 0. Would you like to"
     logFail "restore the system to its original state?"
     if confirmY; then
-        "$DIR"/bin/kurl longhorn rollback-migration-replicas
+        log "Restoring Longhorn replicas to their original state"
+        longhorn_restore_replicas
```

Comment:
```suggestion
        longhorn_restore_migration_replicas
```

Reply:
Oh my, I can't look at this code anymore.